src/status_im2/subs/shell.cljs

```diff
@@ -44,7 +44,7 @@
          ;; https://github.com/status-im/status-mobile/issues/14625
          (= content-type constants/content-type-image)
          {:content-type constants/content-type-image
-          :data         [{:source (resources/get-mock-image :photo2)}]}
+          :data         [{:source (or (:image content) (resources/get-mock-image :photo2))}]}
```

Comment:
Should we remove mock image?

Reply:
I am not positive we covered all the use cases, so I think rather than crashing (or anything), a mock image is better for now, but happy to remove it though

Reply:
yeah, sounds good. Better to keep it for now then.